bootstrap_cfn/mime_packer.py

```diff
@@ -46,7 +46,7 @@ def get_type(content, deftype):
 
 
 def pack(parts, opts={}):
-    outer = MIMEMultipart()
+    outer = MIMEMultipart(boundary='boostrapcfnboundary141')
```

Comment:
I was thinking of using the library's version here but I think I might revert it to something more static. Versions are possible to change and the scripts to remain the same.